From 49bfb032ef8ea9548de71f6082404fe9d5b2736b Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 24 Apr 2017 16:11:53 +0200
Subject: [PATCH 56/73] prefs: disable scrolling animations

One hardware input should correspond to one screen change.
Everything else is a waste of time for the user.
---
 ui/gfx/animation/animation_linux.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git ui/gfx/animation/animation_linux.cc ui/gfx/animation/animation_linux.cc
index 0bcce53e57f33..181c96f9ad7d5 100644
--- ui/gfx/animation/animation_linux.cc
+++ ui/gfx/animation/animation_linux.cc
@@ -13,6 +13,7 @@ namespace {
 // GTK only has a global setting for whether animations should be enabled.  So
 // use it for all of the specific settings that Chrome needs.
 bool AnimationsEnabled() {
+	return false;
   auto* provider = AnimationSettingsProviderLinux::GetInstance();
   return !provider || provider->AnimationsEnabled();
 }
-- 
2.32.0

